src/data/showcases.js

```diff
@@ -1208,6 +1208,15 @@ export const Showcases = [
     source: null,
     tags: ["analytics", "pooltool"],
   },
+  {
+    title: "Peer Review Money",
+    description: 
+      "Pretty straight forward site about the economics of ada, the native currency of the Cardano blockchain.",
+    preview: require("./showcase/peerreview.money.png"),
```

Comment:
yes (your build is failing because) there is no file at this location in your pull request... so you will have to check it into your PR branch & commit.

Reply:
Done